docs/source/notebooks/AR.ipynb

```diff
@@ -16,7 +16,7 @@
   },
```

Comment:
<p>Since you already converted the trace to ArviZ's inference data above, you can just do <code>az.plot_posterior(idata)</code> , this will render faster ;) </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3899/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>